src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
+
+  if (days < 3) {
+    return (days * dayRentPrice);
+  }
+
+  if (days >= 3 && days < 7) {
+    return (days * dayRentPrice) - discount1;
+  }
+
+  return (days * dayRentPrice) - discount2;
```

Comment:
It's not a good idea to use numbers in variable names, what about firstDiscount and secondDiscount, or bigDiscount and smallDiscount